### PR TITLE
EOS-26146 - K8s post-merge 1N and 3N pipeline jobs. 

### PR DESCRIPTION
**Pull request by shailesh-vaidya**
# Problem Statement
- K8s post-merge 1N and 3N pipeline jobs.

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [x] Test Cases cover Happy Path, Non-Happy Path and Scalability - Tested at http://eos-jenkins.colo.seagate.com/job/Cortx-Kubernetes/job/post-merge/view/Kubernetes_Dashboard/
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: jenkins/deployment/k8s_deployment/cortx-cluster.groovy [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    options {
+        timeout(time: 600, unit: 'MINUTES')
+        timestamps()
+        buildDiscarder(logRotator(daysToKeepStr: '20', numToKeepStr: '20'))
+        disableConcurrentBuilds()
+    }
+    environment {
+        CORTX_RE_BRANCH = "kubernetes"
+        CORTX_RE_REPO = "https://github.com/Seagate/cortx-re/"
+        DOCKER_IMAGE_LOCATION = "https://github.com/Seagate/cortx-re/pkgs/container/cortx-all"
```
Comment (gauravchaudhari02) [inline jenkins/deployment/k8s_deployment/cortx-cluster.groovy:17]: Do we need this var?

File: jenkins/deployment/k8s_deployment/cortx-cluster.groovy [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    }
+    parameters {
+        string(name: 'CORTX_RE_BRANCH', defaultValue: 'kubernetes', description: 'Branch or GitHash for Cluster Setup scripts', trim: true)
+        string(name: 'CORTX_RE_REPO', defaultValue: 'https://github.com/Seagate/cortx-re/', description: 'Repository for Cluster Setup scripts', trim: true)
+        string(name: 'CORTX_IMAGE', defaultValue: 'cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-latest-kubernetes', description: 'CORTX-ALL image', trim: true)
+        string(name: 'SNS_CONFIG', defaultValue: '1+0+0', description: 'sns configuration for deployment. Please select value based on disks available on nodes.', trim: true)
+        string(name: 'DIX_CONFIG', defaultValue: '1+0+0', description: 'dix configuration for deployment. Please select value based on disks available on nodes.', trim: true)
+
```
Comment (gauravchaudhari02) [inline jenkins/deployment/k8s_deployment/cortx-cluster.groovy:25]: Can we add a comment here that `hosts` is a manually configured parameter for the job?

File: jenkins/deployment/k8s_deployment/cortx-cluster.groovy (hunk 1 above)
Comment (shailesh-vaidya) [reply jenkins/deployment/k8s_deployment/cortx-cluster.groovy:17]: Yes. As of now, it's only supporting the GitHub registry but we need to extend it for the local registry.

File: jenkins/deployment/k8s_deployment/cortx-cluster.groovy (hunk 2 above)
Comment (shailesh-vaidya) [reply jenkins/deployment/k8s_deployment/cortx-cluster.groovy:25]: Updated comment. Please review.

File: jenkins/deployment/k8s_deployment/cortx-cluster.groovy (hunk 1 above)
Comment (gauravchaudhari02) [reply jenkins/deployment/k8s_deployment/cortx-cluster.groovy:17]: @shailesh-vaidya, could you please comment here that we need to make this var dynamic in future based on either github or local registry?

Comment (shailesh-vaidya) [reply jenkins/deployment/k8s_deployment/cortx-cluster.groovy:17]: Added comments

**Review by gauravchaudhari02 — APPROVED** [review]

**Pull request merged**